lake/top/lake_top.py

```diff
@@ -48,7 +48,8 @@ def __init__(self,
                  config_addr_width=8,
                  remove_tb=False,
                  fifo_mode=True,
-                 add_clk_enable=False):
+                 add_clk_enable=False,
```

Comment:
Did you want to make this true again?
```suggestion
                 add_clk_enable=True,
```

Reply:
I haven't updated the top tests. Since it's clock enable, when you do zero inputs in fault, it will turn the clock enable to low, thus breaking the tests. I will send an update soon.

Reply:
Sounds good